### DISCUSSION TITLE
throwHttpErrors: false and retry functionality

### DISCUSSION THREAD

**Discussion by robertgr991** [sindresorhus/ky#367] [Ideas]
Hello! Right now, the retry functionality is not working if `throwHttpErrors` is `false`.
Would it make sense for you to add another option to switch between retrying or not with `throwHttpErrors: false` with the option being `false` by default to a avoid breaking change?
I know that you can pretty much achieve the same thing with `throwHttpErrors: true` and accessing `err.response`, but it's not explicitly said in the docs that retry doesn't work in that case.

**Comment by szmarczak** [comment]
If you set `throwHttpErrors: false` then it means the response was successful, therefore no need to retry.

**Reply by robertgr991** [reply]
I think it depends on how you interpret this option, at first I saw it as "don't throw an error, just return the response" not as successful vs not successful and I expected the retry to still apply. Maybe you want the response returned but still have the retries applied for some status codes. But I see your point, still I think it should be said in the docs about this.

**Reply by szmarczak** [reply]
PR welcome :)

**Reply by robertgr991** [reply]
Sent a PR here #368
